keras/backend/cntk_backend.py

```diff
@@ -1434,7 +1476,9 @@ def _recurrence(x, states, m):
                 uses_learning_phase = True
 
             if m is not None:
-                new_states = [C.element_select(m, n, s) for n, s in zip(new_states, past_values)]
+                new_states = [C.element_select(m, n, s) for n, s in zip(
+                    new_states,
+                    past_values)]
```

Comment:
Can you make multiple statements for this one too? Thanks!

Reply:
This change has been performed.
Kindly review.